graph/pkg/service/v0/net/headers/headers.go

```diff
@@ -0,0 +1,9 @@
+package headers
+
+const (
+	// "github.com/cs3org/reva/internal/http/services/datagateway" is internal so we redeclare it here
+	// TokenTransportHeader holds the header key for the reva transfer token
+	TokenTransportHeader = "X-Reva-Transfer"
```

Comment:
You can leave `Header` out of the name since the packages is already called `headers`.

Reply:
You could even get rid of the package headers and just have the constants in the `net` package just like `http` has the constants `http.MethodGet` or `http.MethodPost`.

In your case it could be `net.HeaderTokenTransport` and `net.HeaderIfModifiedSince`.